zebra-network/src/protocol/external/addr/in_version.rs

```diff
@@ -24,8 +24,8 @@ use crate::protocol::external::arbitrary::addr_v1_ipv6_mapped_socket_addr_strate
 
 use super::{canonical_socket_addr, v1::ipv6_mapped_socket_addr};
 
-/// A version 1 node address and services, without a last-seen time.
-/// This struct is serialized and deserialized as part of `version` messages.
+/// The format used for Bitcoin node addresses in `version` messages.
+/// Contains a node address and services, without a last-seen time.
```

Comment:
Tiny nit: I'd separate the two lines so that the documentation summary is shorter and the rest of the information is only in the detail section.

```suggestion
/// The format used for Bitcoin node addresses in `version` messages.
///
/// Contains a node address and services, without a last-seen time.
```

Reply:
The different contents of each format are actually really important, so I'd like them to appear in the summary.
(And they're also poorly documented in Bitcoin.)